command/app.go

```diff
@@ -19,6 +19,10 @@ const (
 	appName = "s5cmd"
 )
 
+// AppStorageOptions will be overridden by inner command flags
+// such as if provided `cp -region` will override s5cmd -region flag value.
+var AppStorageOptions storage.StorageOptions
```

Comment:
Instead of keeping global storage options in `command/app.go`, I think it is better to store them in `storage` package. You may pass these options by using` storage.Init(s3opts)`. 